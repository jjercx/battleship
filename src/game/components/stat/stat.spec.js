import React from "react";
import Stat from "./stat";
import renderer from "react-test-renderer";

describe("<Stat />", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<Stat />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
