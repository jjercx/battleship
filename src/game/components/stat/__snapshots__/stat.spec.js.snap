// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stat /> matches snapshot 1`] = `
<div
  className="sc-AxjAm iPIkzj"
>
  <span
    className="sc-AxirZ bibTtL"
  />
  <span
    className="sc-AxiKw cayzsF"
  />
</div>
`;
